File: Javascript/InferenceLocal.js
const axios = require("axios");
const fs = require("fs");

// Model Endpoint
var model_endpoint = "dataset/v";
// Enter your API Key Here
var api_key = "";

const image = fs.readFileSync("YOUR_IMAGE.jpg", {
  encoding: "base64",
});

axios({
  method: "POST",
  url: "https://detect.roboflow.com/" + model_endpoint,
  params: {
    api_key: api_key,
  },
  data: image,
  headers: {
    "Content-Type": "application/x-www-form-urlencoded",
  },
})
  .then(function (response) {
    console.log(response.data);
  })
  .catch(function (error) {
    console.log(error.message);
  });

File: Ruby/InferenceLocal.rb
require 'base64'
require 'httparty'

encoded = Base64.encode64(File.open("YOUR_IMAGE.jpg", "rb").read)
model_endpoint = "dataset/v" # Set model endpoint
api_key = "" # Your API KEY Here

params = "?api_key=" + api_key
+ "&name=YOUR_IMAGE.jpg"

response = HTTParty.post(
    "https://detect.roboflow.com/" + model_endpoint + params,
    body: encoded, 
    headers: {
    'Content-Type' => 'application/x-www-form-urlencoded',
    'charset' => 'utf-8'
  })

  puts response

 